assets/style/about/about.scss

```
@@ -6,9 +6,536 @@
 
 // Author: Nadezhda
 
-// .footer{
-//     &__container{
-//         @extend %default-wrapper;
-//     }
+.main {
+  .main__container {
+    @extend %default-wrapper;
+    @media (max-width: 1650px) {
+      padding-left: 20px;
+      padding-right: 20px;
+    }
+  }
+  &__wr-about-company {
+    background-image: url(../../images/about/bg-moutains.png);
+    min-height: 419px;
+    background-position: center;
+    background-repeat: no-repeat;
+    @media (max-width: $phone-l) {
+      min-height: 550px;
+      background-size: cover;
+    }
+    .main__about-company {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: space-between;
+      max-width: 780px;
+      min-height: 260px;
+      font-family: $font;
+      color: $text-light-color;
+      text-align: center;
+      padding-top: 102px;
+      margin-bottom: 163px;
+      margin-inline: auto;
+      @media (max-width: $laptop-m) {
+        max-width: 90%;
+        min-height: 290px;
+      }
 
-// }
+      .main__about-company-title {
+        font-weight: 700;
+        font-size: 58px;
+        line-height: 55px;
+        @media (max-width: $laptop-m) {
+          font-size: 48px;
+        }
+        @media (max-width: $phone-l) {
+          font-size: 36px;
+          margin-bottom: 10px;
+        }
+        @media (max-width: $phone-m) {
+          font-size: 32px;
+        }
+      }
+      .main__about-company-text {
+        @include default-p($text-light-color);
+        @media (max-width: $tablet-l) {
+          font-size: 18px;
+        }
+        @media (max-width: $phone-l) {
+          font-size: 16px;
+        }
+      }
+    }
+  }
+
+  &__wr-about-company-info {
+    background-color: #e5e5e5;
```

Здесь вроде бы другой цвет по фигме
```suggestion
    background-color: #f1f4f8;
```